new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -109,12 +155,22 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
     // get a unique domain namespace
     logger.info("Getting a unique namespace for WebLogic domain");
     assertNotNull(namespaces.get(1), "Namespace list is null");
-    domainNamespace = namespaces.get(1);
+    domain1Namespace = namespaces.get(1);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain 2");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    domain2Namespace = namespaces.get(2);
 
     // get a unique NGINX namespace
```

Comment:
Do we need to install NGIX and verify application routing in these use cases. We can simply use REST scaling to see if the operator can manage the domain.

Reply:
in this case we do, we want to verify domain still running after operator was uninstalled

Reply:
done